File: DataStructures/C++/stack.cpp
#include<bits/stdc++.h>
using namespace std;

class Stacks{
    public:
    vector<int>arr1 = vector<int>(20);
    int topmost = 0;

    int pushingElement(int y){  
        arr1[topmost] = y;
        cout<<topmost<<"value"<<y<<endl;
        topmost++;
        return 0;
    }

    int popElement(){
        arr1[--topmost];
        cout<<"the number of elements in this array are: "<<topmost<<endl;
        for(int i = 0; i < topmost; i++){
            cout<<arr1[i]<<endl;
        }
        return 0;
    }
};

int main(){
    Stacks s;
    s.pushingElement(5);
    s.pushingElement(4);
    s.pushingElement(3);
    s.pushingElement(2);
    s.popElement();
    s.popElement();
    return 0;
}

File: DataStructures/Java/stack.java
class Stacks{
    int topmost = 0;
    int arr1[] = new int[5];

    void pushing(int x){
        arr1[topmost] = x;
        topmost++;
    }

    void popping(){
        --topmost;
    }

    void printing(){
        for(int i  = 0; i < topmost; i++){
            System.out.println(arr1[i]);
        }
    }
}

class stack{
    public static void main(String args[]){
        Stacks s1 = new Stacks();
        s1.pushing(5);
        s1.pushing(4);
        s1.pushing(3);
        s1.popping();
        s1.popping();
        s1.printing();
    }
}